### ISSUE TITLE
listing compulsory before get_check()

### ISSUE THREAD

**Issue by maelle** [r-hub/rhub#260]
If I only run

```r
rhub::get_check("3be32c1")$cran_summary()
 Error in query("GET GROUP STATUS", list(id = private$group_)) : 
  rhub error 
```

But if I run `rhub::list_my_checks(howmany = 1)` and then `get_check()`, no error.

**Comment by gaborcsardi** [comment]
Full ids are always available without the cache, though.

**Comment by gaborcsardi** [comment]
Closed by #267.
